carmin.yaml

```diff
@@ -521,3 +647,35 @@ definitions:
     properties:
       deleteFiles:
         type: boolean
+  Path:
+    type: object
+    required:
+      - platformURI
+      - lastModificationDate
+      - isDirectory
+    properties:
```

Comment:
We could had an optional md5 or sha1 to allow clients to verify the integrity of the downloads.